README.rst

```diff
@@ -3,23 +3,36 @@
 Motr
 =====
 
-.. Slack Badge  - https://img.shields.io/badge/chat-on%20Slack-blu
-
-.. image:: https://img.shields.io/badge/chat-on%20Slack-blue
-        :target: https://join.slack.com/t/cortxcommunity/shared_invite/zt-femhm3zm-yiCs5V9NBxh89a_709FFXQ?
-        :alt: Slack chat badge
+.. License Badge - https://img.shields.io/badge/License-Apache%202.0-blue.svg
 
+.. image:: https://img.shields.io/badge/License-Apache%202.0-blue.svg
+        :target: https://github.com/Seagate/cortx-motr/blob/main/LICENCE
+        :alt: License Badge
+        
 .. Codacy Badge - https://api.codacy.com/project/badge
 
 .. image:: https://api.codacy.com/project/badge/Grade/e047436e66e54d67b911294ad7fe8b4a
         :target: https://app.codacy.com/gh/Seagate/cortx-motr?utm_source=github.com&utm_medium=referral&utm_content=Seagate/cortx-motr&utm_campaign=Badge_Grade
          :alt: Codacy Badge 
         
-.. License Badge - https://img.shields.io/badge/License-Apache%202.0-blue.svg
+.. Slack Badge  - https://img.shields.io/badge/chat-on%20Slack-blu
+
+.. image:: https://img.shields.io/badge/chat-on%20Slack-blue
+        :target: https://join.slack.com/t/cortxcommunity/shared_invite/zt-femhm3zm-yiCs5V9NBxh89a_709FFXQ?
```

Comment:
Hello @Saumya-Sunder , I see  the top level cortx repo uses this URL for slack.
Should we use it too?
```
https://cortx.link/join-slack
```

Reply:
Sure, that is more up to date than these READMEs. Thanks @huanghua78